svgWriterFx.js

```diff
@@ -163,42 +163,59 @@
         };
         
         this.hasDropShadow = function (ctx) {
-            var omIn = ctx.currentOMNode,
-                dropShadow;
-            if (omIn && omIn.style && omIn.style.fx) {
-                dropShadow = omIn.style.fx.dropShadow;
-            
-                if (dropShadow && dropShadow.enabled) {
-                    return true;
-                }
+            var omIn = ctx.currentOMNode;
+            if (omIn && omIn.style && omIn.style.fx && omIn.style.fx.dropShadowMulti) {
```

Comment:
This seems like you could extract it further to make it easier to drop in support for other languages. Pull this entire thing out to a 'isMultiLayerFxEnabled(omIn, 'dropshadowMulti')